src/main/java/io/wispforest/owo/network/OwoHandshake.java

```diff
@@ -28,16 +31,15 @@
 import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
-import java.util.concurrent.CompletableFuture;
 import java.util.function.ToIntFunction;
 
 @SuppressWarnings("UnstableApiUsage")
 @ApiStatus.Internal
 public final class OwoHandshake {
 
     @SuppressWarnings("unchecked")
-    private static final PacketBufSerializer<Map<Identifier, Integer>> RESPONSE_SERIALIZER =
-            (PacketBufSerializer<Map<Identifier, Integer>>) (Object) PacketBufSerializer.createMapSerializer(Map.class, Identifier.class, Integer.class);
+    private static final Endec<Map<Identifier, Integer>> RESPONSE_SERIALIZER =
+            (Endec<Map<Identifier, Integer>>) (Object) ReflectionEndecBuilder.createMapSerializer(Map.class, Identifier.class, Integer.class);
```

Comment:
This should probably be updated to use a proper MapEndec

Reply:
Possibly